### ISSUE TITLE
story-book axe extension error with MonthText

### ISSUE THREAD

**Issue by toni-sharpe** [toni-sharpe/brainstem-research#52]
### Problem

There's an axe fail in MonthText

`<dl> elements must only directly contain properly-ordered <dt> and <dd> groups, <script>, <template> or <div> elements`

### Solution

It looks like the the `dl` fails with just a `dt` - could just use `ol` here with a title holding the sum figure, this HTML should work without an `if` and include this element
